Modules/Microsoft365DSC/DSCResources/MSFT_EXOTransportConfig/MSFT_EXOTransportConfig.psm1

```diff
@@ -0,0 +1,667 @@
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter()]
+        [System.Boolean]
+        $AddressBookPolicyRoutingEnabled,
+
+        [Parameter()]
+        [System.Boolean]
+        $AllowLegacyTLSClients,
+
+        [Parameter()]
+        [System.Boolean]
+        $ClearCategories,
+
+        [Parameter()]
+        [System.Boolean]
+        $ConvertDisclaimerWrapperToEml,
+
+        [Parameter()]
+        [System.String]
+        $DSNConversionMode,
+
+        [Parameter()]
+        [System.Boolean]
+        $ExternalDelayDsnEnabled,
+
+        [Parameter()]
+        [System.String]
+        $ExternalDsnDefaultLanguage,
+
+        [Parameter()]
+        [System.Boolean]
+        $ExternalDsnLanguageDetectionEnabled,
+
+        [Parameter()]
+        [System.String]
+        $ExternalDsnReportingAuthority,
+
+        [Parameter()]
+        [System.Boolean]
+        $ExternalDsnSendHtml,
+
+        [Parameter()]
+        [System.String]
+        $ExternalPostmasterAddress,
+
+        [Parameter()]
+        [System.String]
+        $HeaderPromotionModeSetting,
+
+        [Parameter()]
+        [System.Boolean]
+        $InternalDelayDsnEnabled,
+
+        [Parameter()]
+        [System.String]
+        $InternalDsnDefaultLanguage,
+
+        [Parameter()]
+        [System.Boolean]
+        $InternalDsnLanguageDetectionEnabled,
+
+        [Parameter()]
+        [System.String]
+        $InternalDsnReportingAuthority,
+
+        [Parameter()]
+        [System.Boolean]
+        $InternalDsnSendHtml,
+
+        [Parameter()]
+        [System.Int32]
+        $JournalMessageExpirationDays,
+
+        [Parameter()]
+        [System.String]
+        $JournalingReportNdrTo,
+
+        [Parameter()]
+        [System.String]
+        $MaxRecipientEnvelopeLimit,
+
+        [Parameter()]
+        [System.Int32]
+        $ReplyAllStormBlockDurationHours,
+
+        [Parameter()]
+        [System.Int32]
+        $ReplyAllStormDetectionMinimumRecipients,
+
+        [Parameter()]
+        [System.Int32]
+        $ReplyAllStormDetectionMinimumReplies,
+
+        [Parameter()]
+        [System.Boolean]
+        $ReplyAllStormProtectionEnabled,
+
+        [Parameter()]
+        [System.Boolean]
+        $Rfc2231EncodingEnabled,
+
+        [Parameter()]
+        [System.Boolean]
+        $SmtpClientAuthenticationDisabled,
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint,
+
+        [Parameter()]
+        [System.String]
+        $CertificatePath,
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $CertificatePassword
+    )
+
+    Write-Verbose -Message "Getting EXOTransportConfig"
```

Comment:
There are no mandatory parameters. So there should be a check that at least one parameter is specified.

Reply:
Added mandatory parameter